testdata/node/machineconfig_nostat.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: custom-seccomp
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - contents:
```

Comment:
@jhou1 what kind of data is stored here?  Does it contain sensitive data?  It's base64 encoded, so it can be easily decoded.

Reply:
The containt of the encoded data is the [seccomp_nostat.json](https://github.com/openshift/verification-tests/blob/155f44d8a0f3df9370aed2254720b59bc1378f8f/testdata/node/seccomp_nostat.json)